core/server/config/env/config.production.json

```diff
@@ -8,5 +8,10 @@
     },
     "paths": {
         "contentPath": "content/"
+    },
+    "logging": {
+        "level": "info",
+        "rotation": true,
+        "transports": ["file"]
```

Comment:
it's not planned to have multiple log files.
in HTTP world, error and request belong together in 99%. 
especially when developers (or we) want to collect logs into a centralised system, it's very helpful to have them together. so each log entry contains: the request, the response and an error if one occurred. This is one entry.